core/relay/docker-compose.yaml

```diff
@@ -56,6 +56,9 @@ services:
     #
     - RELAY_PORT=${RELAY_PORT}
     - RELAY_NAME=${RELAY_NAME}
+    - RELAY_TLS=${RELAY_TLS}
```

Comment:
Can comment these three TLS envs by default, as default behaviour is to use config `toml` file right?
And then in `scripts/convert-compose.sh`, can add to uncomment (in `if` block) and comment these (in `else` block).

Reply:
Yes, done.